scripts/lib/setup/packages.sh

```diff
@@ -202,9 +202,9 @@ function load_rvm_if_available() {
 ###############
 
 function load_nvm_if_available() {
-  [ -f ~/.nvm/nvm.sh ] && source ~/.nvm/nvm.sh
+  [ -f ~/.nvm/nvm.sh ] && chmod +x ~/.nvm/nvm.sh && source ~/.nvm/nvm.sh
```

Comment:
Check if `NVM_DIR` env variable exists and if it does use it when sourcing `nvm.sh`. We set it for Docker containers, and other people can set it themselves if they install NVM in a custom location.